reference/api/rest.rst

```diff
@@ -0,0 +1,358 @@
+|                              |              {                       |              {                      |              {                      |
+|                              |                  "locale": null,     |                  "locale": "en_US", |                 "locale": null,     |
+|                              |                  "scope": null,      |                  "scope": null,     |                 "scope": null,      |
+|                              |                  "data": "mug"       |                  "data": null       |                 "data": "mug"       |
+|                              |              }                       |              }                      |              }                      |
+|                              |          ],                          |          ]                          |          ],                         |
+|                              |          "name": [                   |      }                              |          "name": [                  |
+|                              |             {                        |  }                                  |              {                      |
+|                              |                  "locale": "en_US",  |                                     |                  "locale": "en_US", |
+|                              |                  "scope": null,      |                                     |                  "scope": null,     |
+|                              |                  "data": "Mug"       |                                     |                  "data": null       |
+|                              |             }                        |                                     |              }                      |
+|                              |          ]                           |                                     |          ]                          |
+|                              |      }                               |                                     |      }                              |
+|                              | }                                    |                                     |  }                                  |
++------------------------------+--------------------------------------+-------------------------------------+-------------------------------------+
+
+.. note::
+  For these examples only products values are represented, but usually products also include other information as specified in the standard format.
+
+Response format
+---------------
+
+Get a resource or collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The response format for requests is a JSON object.
+
+Create or update a resource
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When a resource is successfully created or updated, the API returns an HTTP redirection.
+Receiving an HTTP redirection is not an error and clients can follow that redirect if needed.
```

Comment:
It makes sense on creation, but on an update (PUT or PATCH), the resource will not have its URL changed. Or maybe when updating the product identifier ? But this a special case and must be treated as an exception in the documentation.